projects/1.20-fabric/assets/debugify/debugify/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+	"debugify.fix_explanation.mc-123739": "按标识符对配方书条目进行排序。",
+	"debugify.fix_explanation.mc-143474": "重生后生成新的本地玩家时，复制选定的快捷栏槽。",
+	"debugify.fix_explanation.mc-159163": "从实体数据包处理中删除姿势数据。",
+	"debugify.fix_effect.mc-159163": "这将阻止客户端与服务器同步姿势数据，完全由客户端来决定姿势。",
+	"debugify.fix_explanation.mc-162253": "在加载区块时停止客户端光照更新。",
+	"debugify.fix_explanation.mc-165381": "当客户端丢弃物品时取消破坏方块。",
+	"debugify.fix_explanation.mc_176559": "在检查是否应重置破坏方块进程时，不将物品耐久度变化视做物品变更。",
+	"debugify.fix_explanation.mc-197260": "将传递给盔甲架渲染器的光照等级设置为以下四处的最大值：盔甲架下方的方块、盔甲架底部、盔甲架顶部和盔甲架上方的方块。",
+	"debugify.fix_explanation.mc-237493": "在遥测菜单中添加一个选项以完全禁用遥测。通过将所有传出的遥测事件覆盖为未发送的空遥测事件，可以禁用遥测。",
+
+	"debugify.fix_explanation.mc-2025": "由于浮点数精度问题，有时实体的碰撞框大小会比预期稍小。如果实体被推到一起之前这种情况发生了，碰撞框可能会与墙壁交叉。在区块加载时重新计算轴对齐包围盒（AABB）时，因为这些实体会被确定为在墙内，所以会被推动。这个修复程序先向实体的NBT数据中写入一个双精度碰撞框大小的‘AABB’标签，再被加载回来使用。",
+	"debugify.fix_effect.mc-2025": "只有当使用Debugify加载并保存区块后，加载Debugify才能修复问题。",
+	"debugify.fix_explanation.mc-7569": "在每个系统消息的末尾换行。",
+	"debugify.fix_explanation.mc-30391": "对鸡、烈焰人和凋灵进行特判，从而避免产生摔落伤害粒子。",
+	"debugify.fix_explanation.mc-69216": "玩家切换到旁观模式时收回浮漂。",
+	"debugify.fix_explanation.mc-72151": "当狼受到伤害时，有一个函数来调整它所受伤害的量：输入的伤害如果是0则为0.5。此修复程序用于检查伤害值是否为0.5，如果是则其改回0。",
+	"debugify.fix_explanation.mc-88371": "如果返回传送门已被摧毁，则将龙降落的Y值设置为65。",
+	"debugify.fix_explanation.mc-89146": "保存方块实体时严格按照更新的顺序，从而确保方块实体初始化更新也能按照正确的顺序进行。",
+	"debugify.fix_explanation.mc-93018": "只有当狼被驯服后才能开始繁殖。",
+	"debugify.fix_explanation.mc-100991": "向伤害追踪器登记浮标伤害源，与雪球类似。",
+	"debugify.fix_explanation.mc-119417": "在切换到旁观模式时中止玩家睡觉。",
+	"debugify.fix_explanation.mc-119754": "切换至旁观模式时，清除烟花火箭加速鞘翅滑翔的粒子效果。",
+	"debugify.fix_explanation.mc-121706": "拥有远程武器攻击目标的实体将强制目视他们的目标。",
+	"debugify.fix_explanation.mc-121903": "将“LastExecution”标签添加到命令方块矿车的NBT，以保存和加载其上次执行的时间。",
+	"debugify.fix_explanation.mc-124117": "在跨维度传送后，向玩家发送数据包以更新他们的所有效果、经验和能力。",
+	"debugify.fix_explanation.mc-132878": "当护甲受到伤害时会产生破碎粒子。",
+	"debugify.fix_explanation.mc-135971": "覆盖合成槽中的CTRL+Q行为，重复CTRL+Q，直到合成槽中没有剩余物品为止。",
+	"debugify.fix_explanation.mc-155509": "河豚在试图蜇伤玩家之前，先检查自己是否还活着。",
+	"debugify.fix_explanation.mc-160095": "如果仙人掌毗邻方块为移动的活塞，判定掉落条件时检查正在被移动的方块。",
+	"debugify.fix_explanation.mc-179072": "在点燃之前检查苦力怕看到的实体是否是敌人。",
+	"debugify.fix_explanation.mc-183990": "当一个敌对生物进行状态更新时，如其目标已死亡，将不再把该实体作为目标。此修复程序适用于各类群体攻击实体，不止蠹虫。",
+	"debugify.fix_explanation.mc-193343": "如果玩家是旁观者，则移除灵魂疾行。",
+	"debugify.fix_explanation.mc-199467": "修改sin和cos函数的输入，将输入折叠到0~2π的范围，以避免整数溢出。尽管问题汇报只提及客户端，该问题在服务端也存在，因此该修复程序将应用于双端。",
+	"debugify.fix_explanation.mc-200418": "僵尸村民鸡骑士被治愈后不再骑乘在鸡上。",
+	"debugify.fix_explanation.mc-206922": "存在时间小于9刻的物品不会被闪电移除。",
+	"debugify.fix_explanation.mc-215530": "当玩家切换至旁观模式时，重置冰冻时间。",
+	"debugify.fix_explanation.mc-224729": "始终保存ProtoChunk，即使区块保存谓词返回false。",
+	"modmenu.descriptionTranslation.debugify": "修复了在错误跟踪器上发现的Minecraft错误\n\n许可证内容：\nj-Tai的TieFix-根据LGPLv3许可使用的代码\nFlashyReese的Sodium Extra-根据LGPLv3许可使用的代码\nAmpflower的2x2环绕式Saplings修复程序-根据Zlib许可使用的代码"
```

```suggestion
	"modmenu.descriptionTranslation.debugify": "修复了在漏洞追踪器上发现的Minecraft错误\n\n许可证内容：\nj-Tai的TieFix-根据LGPLv3许可使用的代码\nFlashyReese的Sodium Extra-根据LGPLv3许可使用的代码\nAmpflower的2x2树苗周遭判定修复程序-根据Zlib许可使用的代码"
```
[mcw:漏洞追踪器](https://zh.minecraft.wiki/w/漏洞追踪器)

以及，这MC-8187还真有（
看来得找作者补键了，不过不补先合并也行